apps/site/lib/site_web/controllers/facilities_controller.ex

```diff
@@ -0,0 +1,18 @@
+defmodule SiteWeb.FacilitiesController do
+  @moduledoc """
+  Page for display of information about facilities.
+  """
+  use SiteWeb, :controller
+
+  def get_facilities(conn, %{"stop_id" => stop_id}) do
+    res = Facilities.Repo.get_for_stop(stop_id)
+
+    case res do
+      {:error, _x} ->
+        json(conn, res)
```

Comment:
suggestion(non-blocking): we have some helper functions in `SiteWeb.ControllerHelpers` to return error status, they're not widely used but I think `return_error/3` might be a good fit here.